site/content/docs/5.1/migration.md

```diff
@@ -17,23 +17,7 @@ Bootstrap v5.2.0 features a subtle design update for a handful of components and
 
 ### More CSS variables
 
-**We've updated nearly all our components to use CSS variables.** While Sass still underpins everything, each component has been updated to include CSS variables on the component base classes (e.g., `.btn`), allowing for more real-time customization of Bootstrap.
-
-The following components are now built with CSS variables:
-
-- [Alerts]({{< docsref "/components/alerts/" >}})
-- [Badges]({{< docsref "/components/badge/" >}})
-- [Breadcrumbs]({{< docsref "/components/breadcrumb/" >}})
-- [Buttons]({{< docsref "/components/buttons/" >}})
-- [Dropdowns]({{< docsref "/components/dropdowns/" >}})
-- [Navbars]({{< docsref "/components/navbar/" >}})
-- [Pagination]({{< docsref "/components/pagination/" >}})
-- [Popovers]({{< docsref "/components/popovers/" >}})
-- [Progress]({{< docsref "/components/progress/" >}})
-- [Spinners]({{< docsref "/components/spinners/" >}})
-- [Tooltips]({{< docsref "/components/tooltips/" >}})
-
-Read more about CSS variables in each component on their respective documentation pages. The rest of our components, forms, and more will be updated by v5.3.
+**We've updated all our components to use CSS variables.** While Sass still underpins everything, each component has been updated to include CSS variables on the component base classes (e.g., `.btn`), allowing for more real-time customization of Bootstrap. In subsequent releases, we'll continue to expand our use of CSS variables into our layout, forms, helpers, and utilities. Read more about CSS variables in each component on their respective documentation pages.
```

Comment:
Sorry to disturb again the peaceful PRs, but I'm actually wondering if some components weren't tackled yet. For example : Close button, Placeholders, Collapse (not much), Carousel, Button group (not sure about this one)

Reply:
Most of those I'm not sure how best to implement something there. Definitely open to ideas. Carousel could make sense since it already has themes in a way. Button group, only thing I can think there is that border-width isn't much of a CSS variable right now because of some Sass math I think would cause breakages for folks.

Reply:
Ok thanks for your response, I was just asking myself about the wording, I'll try to help on this subject asap !